src/utils/file-type-utils.js

```diff
@@ -0,0 +1,17 @@
+const FORMAT_OVERRIDES = {
```

Comment:
I gotta say I'm not a fan of duplicating the API's logic here, but ya gotta do what ya gotta do.

Reply:
To be fair, this logic DID already exist on the front end (dataset details), in the presentation layer no less. We pulled it  out into a selector and then this file so it was only duplicated once instead of multiple times.